src/utils/cloneNodeHelper.js

```diff
@@ -0,0 +1,48 @@
+import Node from '../Node';
+import Text from '../Text';
+import DOMException from '../DOMException';
+
+function cloneElementNode(element, deep) {
+    let clone = element.ownerDocument.createElement(element.nodeName);
+    clone._ownerDocument = null;
+    clone._parentNode = null;
+    let attributes = element.attributes;
+    for (let key in attributes) {
+        if (attributes.hasOwnProperty(key)) {
+            clone.setAttribute(key, element.getAttribute(key));
+        }
+    }
+
+    if (deep) {
+        element.childNodes.every((child) => {
+            clone.appendChild(cloneNode(child, deep));
+            return true;
+        });
+    }
+
+    return clone;
+}
+
+function cloneTextNode(element) {
+    return new Text(element.value);
+}
+
+/**
+ * Function return copy of element and if request deep copy of element
+ *
+ * @param {Node} element
+ * @param {boolean} deep
+ * @ignore
+ */
+function cloneNode(element, deep) {
```

Comment:
export function cloneNode ?

for libs/helpers/tools prefer export functions and not as default